samples/apps/copilot-chat-app/webapp/src/components/chat/ChatInput.tsx

```diff
@@ -79,15 +79,15 @@ export const ChatInput: React.FC<ChatInputProps> = (props) => {
         async function initSpeechRecognizer() {
             const speechService = new SpeechService(process.env.REACT_APP_BACKEND_URI as string);
 
-            var response = await speechService.validSpeechKeyAsync();
+            var response = await speechService.validSpeechKeyAsync(await AuthHelper.getSKaaSAccessToken(instance, inProgress));
```

Comment:
not related to this PR, rename to "validateSpeechKeyAsync"

Reply:
done